include/wx/vector.h

```diff
@@ -519,6 +519,13 @@ class wxVector
         const size_t idx = it - begin();
         const size_t after = end() - it;
 
+        // Unfortunately gcc 12 still complains about use-after-free even if
```

Comment:
```suggestion
        // Unfortunately gcc 12 still complains about use-after-free even though
```

Reply:
Thanks, will apply this (and the other typo fix) when merging.